Postman/dex.postman_collection.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Not familiar with postman tests, can't review this. I'll make sure to look into it for the future